.github/workflows/ci.yml

```diff
@@ -106,3 +106,52 @@ jobs:
 
       - name: Test
         run: cargo test --all-features
+
+  test-api:
+    name: Test Weechat API
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        version:
+          - v3.8
+          - v4.0.0
+          - v4.1.0
+          - v4.2.0
+          - v4.3.0
+          - v4.4.0
+          - v4.5.0
```

Comment:
Very nice that you set up the CI to test a wide array of versions.

Reply:
Definitely helped me with polishing the PR, as I've misremembered incompatible versions (thought 4.0.0/4.1.0, was 4.1.0/4.2.0) and testing it locally is slightly annoying with having to checkout each version either in same repo or each version into different path